<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyPd</class>
 <widget class="QDialog" name="ModifyPd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ModifyPd{
	background-color:rgb(244, 244, 244);
	color: black;
	border: 1px solid #DDDDDD;
}
</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#titleLabel{
background-color: #F8F8F8;
padding-left: 16px;
color: #222222;
font-weight: bold;
}</string>
   </property>
   <property name="text">
    <string>新密码设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="tipLabel">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>56</y>
     <width>408</width>
     <height>56</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#tipLabel{
background-color: rgba(62, 206, 254, 0.10);
border: 1px solid rgba(62, 206, 254, 0.50);
border-radius: 4px;
font-size: 12px;
color: #222222;
padding: 10px 12px 10px 12px;
}</string>
   </property>
   <property name="text">
    <string>密码可由数字，大写英文字母，小写英文字母，特殊字符组成，至少包含两种类型，长度要求8-16字符</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>326</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QLineEdit" name="newPdEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>258</width>
        <height>38</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>258</width>
        <height>38</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">#newPdEdit{
background-color:#FFFFFF;
color: black;

}</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="newPd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>2</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">#newPd{
	font-family:'微软雅黑';
	font-size: 14px;
	font-weight: bold;
	color: #000000;
}</string>
      </property>
      <property name="text">
       <string>新密码</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="newPdAgain">
      <property name="styleSheet">
       <string notr="true">#newPdAgain{
	font-family:'微软雅黑';
	font-size: 14px;
	font-weight: bold;
	color: #000000;
}</string>
      </property>
      <property name="text">
       <string>再次输入</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="newPdAgainEdit">
      <property name="minimumSize">
       <size>
        <width>258</width>
        <height>38</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>258</width>
        <height>38</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#FFFFFF;
color: black;
</string>
      </property>
      <property name="text">
       <string>再次输入新密码...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="submitBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>264</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#submitBtn{
background-color: #3ECEFE;
border: none;
border-radius: 4px;
color: #FFFFFF;
font-size: 14px;
}</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>264</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#cancelBtn{
background-color: #FFFFFF;
border: 1px solid #DDDDDD;
border-radius: 2px;
color: black;
font-size: 14px;
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>281</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#messageLabel{
	font-size: 12px;
	color: red;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
